server/routerlicious/packages/routerlicious-base/src/riddler/api.ts

```diff
@@ -72,12 +78,39 @@ export function create(
         returnResponse(storageP, response);
     });
 
+    /**
+     * Updates the details for the given tenant based on instructions in the request body
+     */
+    router.patch("/tenants/:id", (request, response) => {
```

Comment:
I would also split this into two. One for key and one for customData. And follow the same return chararecteristics of update storage and orderer.

Reply:
That makes sense. In this implementation I hadn't been treating key and customData as sub-resources, just as properties. I'll treat them as such from now on 😄